doc/source/examples/metrology/Creating Network Definitions for SOLT Calibration Standards.ipynb

```diff
@@ -0,0 +1,1116 @@
+    ")\n",
+    "line_open = medium_open.line(\n",
+    "    d=1, unit='m',\n",
+    "    z0=medium_open.Z0, embed=True\n",
+    ")\n",
+    "\n",
+    "medium_short = DefinedGammaZ0(\n",
+    "    frequency=freq,\n",
+    "    gamma=gamma_l_short, Z0=zc_short, z0=50\n",
+    ")\n",
+    "line_short = medium_short.line(\n",
+    "    d=1, unit='m',\n",
+    "    z0=medium_short.Z0, embed=True\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Modeling the Shunt Impedance\n",
+    "\n",
+    "Then, we need to model the shunt impedance of the open and short standards.\n",
+    "For the open standard, it's a capacitance. For the short standard, it's\n",
+    "an inductance.\n",
+    "\n",
+    "Both are modeled as a third-degree polynomial, as a function of frequency.\n",
+    "In `numpy`, one can quickly define such a function via\n",
+    "`np.poly1d([x3, x2, x1, x0])`. This is a higher-order function which accepts\n",
+    "a list of coefficients in descending order, and returns a callable polynomial\n",
+    "function.\n",
+    "\n",
+    "After the polynomial is evaluated, we can generate the frequency-dependent\n",
+    "capacitors and inductors. The open circuit is modeled as a `medium.shunt_capacitor()`\n",
+    "followed by an ideal `medium.open()` The short circuit is modeled as a\n",
+    "series `medium.inductor()` followed by an ideal `medium.short()` (not `medium.shunt_inductor()`).\n",
+    "\n",
+    "Because the capacitor and inductor are defined with respect to the port impedance,\n",
+    "not any particular lossy transmission line, to avoid confusions, we use `ideal_medium`,\n",
+    "not `medium_open` or `medium_short` in the following examples (although the latter\n",
+    "two are usable, they also use the port impedance)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# use ideal_medium, not medium_open and medium_short to avoid confusions.\n",
+    "\n",
+    "capacitor_poly = np.poly1d([\n",
+    "    -0.1597 * 1e-45,\n",
+    "    23.17   * 1e-36,\n",
+    "    -310.1  * 1e-27,\n",
+    "    49.43   * 1e-15\n",
+    "])\n",
+    "capacitor_list = capacitor_poly(freq.f)\n",
+    "shunt_open = ideal_medium.shunt_capacitor(capacitor_list) ** ideal_medium.open()\n",
+    "\n",
+    "inductor_poly = np.poly1d([\n",
+    "    -0.01   * 1e-42,\n",
+    "    2.171   * 1e-33,\n",
+    "    -108.5  * 1e-24,\n",
+    "    2.077   * 1e-12\n",
+    "])\n",
+    "inductor_list = inductor_poly(freq.f)\n",
+    "shunt_short = ideal_medium.inductor(inductor_list) ** ideal_medium.short()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `medium.open()` termination is important, a line terminated solely by a\n",
+    "`shunt_capacitor()` without an open produces invalid S-parameters due to\n",
+    "implementation-related reasons."
```

Comment:
open termination is needed because shunt capacitor creates a two-port and the other port needs to be open. Series capacitor terminated with short could have also been used to get the same result.

Reply:
Just added your clarification.